### DISCUSSION TITLE
Sky childrens of the light new update

### DISCUSSION THREAD

**Discussion by Malik993** [thatskymod/Sky-CotL-Scripts#67] [Q&A] [answered]
Today I update my sky light and I notice that canvas scripts like smite and dakun langit is not working properly some of them is working but not like before. Please fix it as soon as possible thanks we are waiting for your response. My email address is.. zeamalik99@gmail.com.

**Comment by littleghostav** [comment]
Oh god, I thought only my Dakun had a malfunction, I was desperate thinking there was something wrong with my sky

**Comment by Malik993** [comment]
Hey. Are you still facing same problem or is it fix now

**Comment by thatskymod** [comment, marked as answer]
@Malik993, kindly note that you closed your own thread. As a result, you are less likely to receive a response.

To address your question:

The session error you were experiencing has now been fixed in [Canvas/Libs/libLangit020rc.so](https://github.com/thatskymod/Sky-CotL-Scripts/blob/main/Canvas/Libs/libLangit021rc.so).

Thank you for your patience while the issue was being worked on.

**Reply by Ksisheheh11221** [reply]
Làm ơn hãy giải quyết nhanh chóng

**Reply by thatskymod** [reply]
@Ksisheheh11221 I am sorry, but it looks like the question in this thread has already been sufficiently addressed, with nothing left to resolve. Could you please specify the exact problem you are facing? We'll be glad to look into it further. Thank you!
